### PR TITLE
chore: Update Fleek url

### PR DESCRIPTION
**Pull request by vesahc**
Fleek deployment was migrated and URL has changed to https://ipfs-public-gateway-check.on.fleek.co/  

Note that for future migrations it may be better to change the existing URL first and claim it on the new deployment vs. deleting the old. The name seems to get stuck in the db when deleted.

### CODE REVIEW

**Comment by 2color** [comment]
Thanks for updating this. Is there any way to reclaim the old URL? There are links lurking around pointing to it and it has an outdated version of the deployment

**Comment by vesahc** [comment]
Yeah it's strange that it's still active after it was deleted. Waiting for a response now to see if they can help us out.

**Comment by vesahc** [comment]
Ok old URL is gone and confirmed purged from their db's, we're still having trouble reclaiming it though. They're going to  ask their devs if there's anything else they can do to free it up, but it's not looking very promising.

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 View the Public Gateway Checker in action
 
-- on Fleek: <https://ipfs-public-gateway-checker.on.fleek.co/>
-- on GitHub Pages: <https://ipfs.github.io/public-gateway-checker/>
+* on Fleek:  https://ipfs-public-gateway-check.on.fleek.co/
```
Comment (2color) [inline README.md:7]:
```suggestion
* on Fleek: https://ipfs-public-gateway-check.on.fleek.co/
```

**Review by 2color — APPROVED** [review]
Better to have a working url for now.

**Pull request merged**